blogposts/2021/release-3.25.md

```diff
@@ -27,6 +27,8 @@ changelogItems:
   - description: "When previewing a campaign spec, changesets can be filtered by current state or the action(s) to be performed."
     url: https://github.com/sourcegraph/sourcegraph/issues/16960
     category: Campaigns
-
+  - description: "Syncing Perforce depots directly via Sourcegraph UI is supported behind the feature flag `"experimentalFeatures": { "perforce": "enabled" }`, for more information, see [how to add your Perforce depots](https://docs.sourcegraph.com/admin/repo/perforce)."
```

Comment:
```suggestion
  - description: "Syncing Perforce depots directly via Sourcegraph UI is supported behind the feature flag `"experimentalFeatures": { "perforce": "enabled" }`. For more information, see [how to add your Perforce depots](https://docs.sourcegraph.com/admin/repo/perforce)."
```

Reply:
@abschu small suggestion to break this into two sentences.

Reply:
An alternate suggestion (made the same suggestion to the changelog PR):

```suggestion
  - description: "Experimental: Sync Perforce depots directly through the Sourcegraph UI. To enable, use the feature flag `"experimentalFeatures": { "perforce": "enabled" }`. For more information, see <a href=\"https://docs.sourcegraph.com/admin/repo/perforce\">how to add your Perforce depots."
```